docs/UserGuide.md

```diff
@@ -203,15 +204,31 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Filtering based on selected attributes: `filter`
+
+Filters out the contacts that satisfy the requirements.
+You can choose to filter by day available, by name or by tags.
+
+Format: 
+1. `filter tag TAG_NAME`
+2. `filter name PERSON_NAME`
+3. `filter day DAY`
```

Comment:
Also by our user guide definition remember we can use `...` to indicate that the keyword can be used multiple times. So you can put `filter tag TAG_NAME...`

Reply:
Alright included it in the UG!